desk/app/diary.hoon

```diff
@@ -144,29 +149,26 @@
   --
 ::
 ++  watch-epic
-  |=  her=ship
+  |=  [her=ship leave=?]
   ^+  cor
   =/  =wire  /epic
   =/  =dock  [her dap.bowl]
-  ?:  (~(has by wex.bowl) [wire dock])
-    cor
-  (emit %pass wire %agent [her dap.bowl] %watch /epic)
+  =?  cor  leave  (emit %pass wire %agent dock %leave ~)
+  (emit %pass wire %agent dock %watch /epic)
 ::
 ++  take-epic
   |=  =sign:agent:gall
   ^+  cor
   ?+    -.sign  cor
       %kick
-    (watch-epic src.bowl)
+    (watch-epic src.bowl |)
   ::
       %fact
     ?.  =(%epic p.cage.sign)
       ~&  '!!! weird fact on /epic'
       cor
     =+  !<(=epic:e q.cage.sign)
-    ?.  =(epic okay:d)
-      cor
-    ~&  >>  "good news everyone!"
+    ~&  >>  "resolving epic change, current: {<okay:d>} received: {<epic>}"
```

Comment:
Are we ok with printing this also in the case where current & received are identical? If we don't care for that, `~?` might be better.

Also, do we _really_ need the priority on this printf?

Reply:
no we don't probably lol